cypress/e2e/tests/upgrade/after_upgrade.test.ts

```diff
@@ -200,17 +201,16 @@ describe(["@post-upgrade"], "Performing post-upgrade validations", () => {
 
     it("Enabling RWX, validating it works, disabling it", function () {
         MavenConfiguration.open();
-        let rwxEnabled = false;
-        isEnabled(clearRepository, rwxEnabled);
-
-        rwxEnabled = true;
-        patchTackleCR("configureRWX", rwxEnabled);
-        MavenConfiguration.open();
-        isEnabled(clearRepository, rwxEnabled);
-
-        rwxEnabled = false;
-        patchTackleCR("configureRWX", rwxEnabled);
-        MavenConfiguration.open();
-        isEnabled(clearRepository, rwxEnabled);
+        isRwxEnabled().then((rwxEnabled) => {
+            expect(rwxEnabled).to.be.false;
+            isEnabled(clearRepository, rwxEnabled);
+            rwxEnabled = true;
+            patchTackleCR("configureRWX", rwxEnabled);
```

Comment:
I have noticed that after patching the tackle CR, it helps to reload the page

Reply:
it works fine for me as is.

Reply:
`patchTackleCr()` runs `cy.reload();` at the end, so page is reloaded. No need to do this twice.